src/Zomp.SyncMethodGenerator/SyncMethodSourceGenerator.cs

```diff
@@ -25,162 +27,161 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
         context.RegisterPostInitializationOutput(ctx => ctx.AddSource(
             $"{CreateSyncVersionAttribute}.g.cs", SourceText.From(SourceGenerationHelper.CreateSyncVersionAttributeSource, Encoding.UTF8)));
 
-        IncrementalValuesProvider<MethodDeclarationSyntax> methodDeclarations = context.SyntaxProvider
+        var disableNullable =
+            context.CompilationProvider.Select((c, _)

```

Comment:
@GerardSmit, if you're good with this change, I'll merge.

Following this [example](https://github.com/AndyAyersMS/runtime/blob/0f2cf16467dbd376bf4ac3040262699f850a9155/src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/ComClassGenerator.cs#L22) as well as this [section](https://andrewlock.net/creating-a-source-generator-part-9-avoiding-performance-pitfalls-in-incremental-generators/#5-be-careful-using-compilationprovider) of the tutorial.

Reply:
Looks good 👍

The unit test is still succeeding, so the caching still works. 😄